perception/image_projection_based_fusion/src/roi_cluster_fusion/node.cpp

```diff
@@ -232,12 +236,14 @@ void RoiClusterFusionNode::fuseOnSingleImage(
       }
     }
     debug_image_rois.push_back(feature_obj.feature.roi);
+    debug_max_iou_for_image_rois.push_back(max_iou);
```

Comment:
How about this?
```suggestion
    debugger_->max_iou_for_image_rois_.push_back(max_iou);
```

Reply:
Other parts also do the same trick. 

```
debug_image_rois.push_back(feature_obj.feature.roi);
```

Would you like to refactor all these statement?

``` cpp
// following part also does copy vector to vector
    debugger_->image_rois_ = debug_image_rois;
    debugger_->obstacle_rois_ = debug_pointcloud_rois;
    debugger_->obstacle_points_ = debug_image_points;
    debugger_->max_iou_for_image_rois_ = debug_max_iou_for_image_rois;
```

Reply:
Other parts also do the same trick. 

```
debug_image_rois.push_back(feature_obj.feature.roi);
```

Would you like to refactor all these statement?

``` cpp
// following part also does copy vector to vector
    debugger_->image_rois_ = debug_image_rois;
    debugger_->obstacle_rois_ = debug_pointcloud_rois;
    debugger_->obstacle_points_ = debug_image_points;
    debugger_->max_iou_for_image_rois_ = debug_max_iou_for_image_rois;
```

Reply:
@yukkysaito 
I found your suggestion is not safe because `debugger_` object does not exist if `debug mode` is not set to `true`.
This is why other debug topics are directory copied from temporary vector.

Reply:
@YoshiRi I do not want to put debugging code in the normal code as much as possible. :pray: 
https://github.com/autowarefoundation/autoware.universe/pull/5541/files#diff-6ae5789ac3727b6d80a8069bc0bb0ed78960010659def7095755aa8a169080cbR89-R90

>I found your suggestion is not safe because debugger_ object does not exist if debug mode is not set to true.
This is why other debug topics are directory copied from temporary vector.

How about this?
```
debugger_?debugger_->max_iou_for_image_rois_.push_back(max_iou):;
```

Reply:
@yukkysaito 
Make sense. Fixed in https://github.com/autowarefoundation/autoware.universe/pull/5541/commits/887eca8c88d44c239f51ab4a9576b96b91a175d2.